src/main/java/frc/robot/commands/GrabberLifterCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands;
+import frc.robot.subsystems.GrabberLifter;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+public class GrabberLifterCommand extends CommandBase {
+    private final GrabberLifter m_GrabberLifter;
+    private double Speed;
+  
+    public GrabberLifterCommand(double speed, GrabberLifter subsystem) {
+  
+      m_GrabberLifter = subsystem;
+      double Speed = speed;
+      
+  
+      //addRequirements(m_Drivetrain);
+  
+      // Use addRequirements() here to declare subsystem dependencies
+      
+    }
+  
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+  
+      m_GrabberLifter.lift(Speed);
+      isFinished();
+  
+  
+    }
+  
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+  
+  
+  
+  
+    }
+  
+    // Called once the command ends or is interrupted.
+    @Override
+    public void end(boolean interrupted) {
+  
+    }
```

Comment:
add this as a new method below (sorry didn't realize this was automatically added)
this will automatically end the command the first time it is run after initialize
```
 @Override
  public boolean isFinished() {
    return true;
  }
```


(source; https://github.com/Rundle-Robotics/2023-Charged-Up/blob/6c0f9dd39d4c51a59897b680f9d681ebdee34850/src/main/java/frc/robot/commands/ExampleCommand.java#L39)